tests/samplers_tests/test_grid.py

```diff
@@ -37,7 +37,7 @@ def objective(trial: Trial) -> float:
     search_space = {
         "b": np.arange(-0.1, 0.1, 0.05),
         "c": ("x", "y", None, 1, 2.0),
-        "d": [-0.5, 0.5],
+        "d": [5.0, 5.0],
```

Comment:
I'm not sure, but do you intentionally set the same value (i.e., `5.0`)? Or is it a typo for `-5.0, 5.0`?

Reply:
This was a typo. Thank you for letting me know this. I've fixed it.

Reply:
Thank you!